index.bs

```diff
@@ -11266,54 +11266,101 @@ task queue=] of its associated {{BaseAudioContext}}.
+
+				1. Let |completion| be an uninitialized variable.
+
+				1. [=Prepare to run script=] with the [=current settings object=].
+
+				1. [=Prepare to run a callback=] with the [=current settings object=].
+
+				1. Let |getResult| be
+					<a href="https://tc39.github.io/ecma262/#sec-get-o-p">
+					Get</a>(|O|, "process").
+
+				1. If |getResult| is an
+					<a href="https://tc39.github.io/ecma262/#sec-completion-record-specification-type">
+					abrupt completion</a>, set |completion| to |getResult| and jump to the step
+					labeled <a href="#audio-worklet-render-return">return</a>.
+
+				1. Set |processCallback| to |getResult|.\[[Value]].
+
+				1. If <a href="https://tc39.es/ecma262/#sec-returnifabrupt-shorthands">!</a>
+					<a href="https://tc39.es/ecma262/#sec-iscallable">
+					IsCallable</a>(|processCallback|) is `false`, then:
+
+					1. Set |completion| to new
+						<a href="https://tc39.es/ecma262/#sec-completion">Completion</a>
+						{\[[Type]]: throw, \[[Value]]: a newly created
+						<a href="https://tc39.es/ecma262/#sec-native-error-types-used-in-this-standard-typeerror">
+						TypeError</a> object, \[[Target]]: empty}.
+
+					1. Jump to the step labeled
+						<a href="#audio-worklet-render-return">return</a>.
+
+				1. Set {{[[callable process]]}} to `true`.
+
+				1. Perform the following substeps:
+
+					1. Let |args| be a
+						<a href="https://heycam.github.io/webidl/#web-idl-arguments-list">
+						Web IDL arguments list</a> consisting of
+						{{AudioWorkletProcessCallback/inputs}},
+						{{AudioWorkletProcessCallback/outputs}}, and
+						{{AudioWorkletProcessCallback/parameters}}.
+
+					1. Let |esArgs| be the result of
+						<a href="https://heycam.github.io/webidl/#web-idl-arguments-list-converting">
+						converting</a> |args| to an ECMAScript arguments list.
+
+					1. Let |callResult| be the <a href="https://tc39.github.io/ecma262/#sec-call">
+						Call</a>(|processCallback|, |O|, |esArgs|). This operation
+						[=Computing a block of audio|computes a block of audio=] with |esArgs|.
+						Upon a successful function call, a buffer containing copies of
+						the elements of the {{Float32Array}}s passed via the
+						{{AudioWorkletProcessCallback/outputs}} is
+						[=Making a buffer available for reading|made available for reading=].
+						Any {{Promise}} resolved within this call will be queued into the
+						microtask queue in the {{AudioWorkletGlobalScope}}.
+
+					1. If |callResult| is an
+						<a href="https://tc39.github.io/ecma262/#sec-completion-record-specification-type">
+						abrupt completion</a>, set |completion| to |callResult| and jump to the
+						step labeled <a href="#audio-worklet-render-return">return</a>.
+
+					1. Set |processor|’s <a>active source</a> flag to
+						<a href="https://tc39.github.io/ecma262/#sec-toboolean">
+						ToBoolean</a>(|callResult|.\[[Value]]).
+
+				1. <i id="audio-worklet-render-return">Return:</i> at this point |completion|
```

Comment:
Can you use something other than italics here?  It makes it hard to distinguish between algorithm vars (also in italics).  Maybe bold?  Or smallcaps or something not italics.